### PR TITLE
Clean up governance YAML files

### PR DESCRIPTION
**Pull request by LEARAX**
This PR addresses #499, and attempts to clean up many inconsistencies/issues present in the content of the YAML files relating to the governance page. It fixes a few unescaped characters, unnecessary quotations, and some inconsistent capitalization and period usage, among a few other things.

### CODE REVIEW

**Comment by LEARAX** [comment]
@ashleygwilliams My concern is that then some descriptions (which aren't complete sentences) will have capitalization and terminating periods, while other descriptions won't. Is this acceptable?

**Comment by ashleygwilliams** [comment]
@LEARAX i think i would eventually like them all to be complete sentences :) if you are interested in doing that- that would be very welcome! but if not i think we can use capitalization and periods, and work to iteratively make them complete sentences.

**Comment by LEARAX** [comment]
@ashleygwilliams Is this acceptable?

File: src/data/teams/alumni/team.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
 route: alumni
 name: Rust team alumni
-description: "enjoying a leisurely retirement"
+description: enjoying a leisurely retirement
```
Comment (nrc) [inline src/data/teams/alumni/team.yml]: This looks like it still needs capitalisation.

File: src/data/teams/community/team.yml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 route: community
-name: Community Team
-description: coordination and supporting events, content creation, the RustBridge program, and the survey
+name: Community team
+description: coordinating and supporting events, creating content, managing the RustBridge program, and overseeing the survey
```
Comment (nrc) [inline src/data/teams/community/team.yml]: this one too

File: src/data/teams/core/team.yml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 route: core
 name: Core team
-description: Overall direction and policies of the project, subteam leadership, cross-cutting concerns.
-email: "core@rust-lang.org"
+description: directing and managing project policies overall, leading subteams, and cross-cutting concerns
```
Comment (nrc) [inline src/data/teams/core/team.yml]: and here

Comment (nrc) [reply src/data/teams/core/team.yml]: I'd prefer skipping "overall" here

File: src/data/teams/crates-io/team.yml [hunk 4]
```diff
@@ -1,6 +1,6 @@
 route: crates-io
 name: Crates.io team
-description: management of operations, development, and policies for crates.io
-chat: Infra #crates-io channel on Discord
+description: managing operations, development, and policies for crates.io
```
Comment (nrc) [inline src/data/teams/crates-io/team.yml]: here

File: src/data/teams/dev-tools/wgs/debugging.yml [hunk 5]
```diff
@@ -1,5 +1,5 @@
 route: debugging
 name: Debugging working group
 description: debugging tools and foundations
-chat: "Dev-Tools Team #wg-debugging on Discord"
```
Comment (nrc) [inline src/data/teams/dev-tools/wgs/debugging.yml]: The "Team" here and in others is in the name of the category of channels on Discord. We probably want to leave it in, but maybe add a comma or something

File: src/data/teams/language-and-compiler/wgs/codegen.yml [hunk 6]
```diff
@@ -1,5 +1,5 @@
 route: codegen
-name: Codegen
-description: Improving code generated by rustc.
-chat: #wg-codegen on IRC
+name: Codegen team
```
Comment (nrc) [inline src/data/teams/language-and-compiler/wgs/codegen.yml]: These compiler groups should be "working group"s, not "team"s

**Comment by LEARAX** [comment]
> There's still a lot of capitalisation issues with descriptions

@nrc I agree – I think all the descriptions should start with a lowercase letter and not be period-terminated. However, if you think they should start with capital letters, I'll update this to be consistent with that.

**Comment by nrc** [comment]
> I agree – I think all the descriptions should start with a lowercase letter and not be period-terminated. However, if you think they should start with capital letters, I'll update this to be consistent with that.

I definitely think they should all be consistent. Most of the website uses proper sentence punctuation, so I think it would be good to do so here too.

**Comment by pietroalbini** [comment]
Hi! The data moved to https://github.com/rust-lang/team repository, so I'm closing this. It would be awesome if you could make the changes there!

**Pull request closed without merging**